sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlattenTest.java

```diff
@@ -121,16 +121,30 @@ public void testFlattenPCollectionListEmpty() {
   public void testFlattenInputMultipleCopies() {
     Pipeline p = TestPipeline.create();
 
-    PCollection<String> lines = p.apply("mkLines", Create.of(LINES));
-    PCollection<String> lines2 = p.apply("mkOtherLines", Create.of(LINES2));
-
-    PCollection<String> flattened = PCollectionList.of(lines)
-        .and(lines2)
-        .and(lines)
-        .and(lines)
-        .apply(Flatten.<String>pCollections());
-
-    PAssert.that(flattened).containsInAnyOrder(Iterables.concat(LINES, LINES, LINES, LINES2));
+    PCollection<Long> longs = p.apply("mkLines", CountingInput.upTo(10));
+    PCollection<Long> biggerLongs =
+        p.apply("mkOtherLines", CountingInput.upTo(10))
```

Comment:
No action required, but having a starting point seems nicer than mapping over it. This test is a demonstration that it is useful. General goal: multiple PCollections automatically generated with disjoint elements.